lib/node_modules/@stdlib/math/base/special/cidentity/lib/native.js

```diff
@@ -0,0 +1,57 @@
+/**
+* @license Apache-2.0
+*
+* Copyright (c) 2023 The Stdlib Authors.
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*    http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+'use strict';
+
+// MODULES //
+
+var Complex128 = require( '@stdlib/complex/float64' );
+var addon = require( './../src/addon.node' );
+
+
+// MAIN //
+
+/**
+* Evaluates the identity function for a double-precision complex floating-point number.
+*
+* @param {Complex128} z - input value
+* @returns {Complex128} input value
+*
+* @example
+* var Complex128 = require( '@stdlib/complex/float64' );
+* var real = require( '@stdlib/complex/real' );
+* var imag = require( '@stdlib/complex/imag' );
+*
+* var v = cidentity( new Complex128( -1.0, 2.0 ) );
+* // throws <ReferenceError>
+*
+* var re = real( v );
+* // throws <TypeError>
+*
+* var im = imag( v );
+* // throws <TypeError>
```

Comment:
The return values are incorrect. Likely need to ensure that the add-on is compiled when committing, due to lint auto-fixing. 

cc @Planeshifter

Reply:
@steff456 These return values do not seemed to have been fixed.

Reply:
I'm not sure what is happening, I change them and when I go ahead and do a commit with the `make commit` command it changes to these values... 

I have the addon already compiled 😕

Reply:
Update: I created the commit with the `--no-verify` flag, and this is the only PR where I had this issue. I'll check in future PRs if the behavior repeats.

Reply:
Once merged, I'll pull down and try to compile on my end.